horace_core/sqw/@d1d/d1d.m

```diff
@@ -30,5 +30,32 @@
         %            %
         %               obj = sqw(S);
         %        end
+        function obj = loadobj(S)
+            % Load a sqw object from a .mat file
+            %
+            %   >> obj = loadobj(S)
+            %
+            % Input:
+            % ------
+            %   S       An instance of this object or struct
+            %
+            % -------
+            % Output:
+            %   obj     An instance of this object
+            obj = d1d(S);
```

Comment:
not sure this is useful: the object case is covered below (for one and array arg) and the struct case below that (one and array cases)